### PR TITLE
Switch to better-maintained node-protobuf module

### PR DESCRIPTION
**Pull request by lovell**
As discussed in #7, plus I've added installation instructions for the common package managers.

I've made a minor version bump for now, but we may want to do a major version bump for this.

I've also taken the liberty of creating a Travis CI config to install Riemann and run the tests, e.g. https://travis-ci.org/lovell/riemann-nodejs-client/builds/83315017

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,14 @@
 {
   "name": "riemann",
-  "version": "0.5.1",
+  "version": "0.6.0",
```
Comment (perezd) [inline package.json]: Its time for 1.0....hopefully our users are using versioning properly, so they won't be as negatively impacted potentially.

**Comment by perezd** [comment]
LGTM, lets go with a 1.0 release here.

**Comment by lovell** [comment]
Great, job done, thank you. https://travis-ci.org/lovell/riemann-nodejs-client/builds/83431801

**Comment by perezd** [comment]
I'll do a release tonight, thank you for your contributions!

**Comment by perezd** [comment]
This has been published to npm. Thank you very much for your contributions!

**Pull request merged**